platform.js

```diff
@@ -279,6 +279,18 @@ module.exports = {
       }
     });
   },
+  mkdirRecursively(path) {
+    return new Promise((resolve, reject)=>{
+      module.exports.callMkdirp(path, (err)=>{
+        if (!err) {
+          resolve();
+        } else {
+          log.error("Error creating directory", "Error creating directory: " + path);
+          reject({ message: "Error creating directory", userFriendlyMessage: "Error creating directory", error: err });
```

Comment:
@fjvallarino it might be useful to append the directory path to the error messages